Directory.Build.props

```diff
@@ -19,10 +19,8 @@
     <SourceDir>$(ProjectDir)src\</SourceDir>
 
     <!-- Output directories -->
-    <BinDir Condition="'$(BinDir)'==''">$(ProjectDir)bin/</BinDir>
-    <ObjDir Condition="'$(ObjDir)'==''">$(BinDir)obj/</ObjDir>
-    <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'==''">$(ObjDir)</BaseIntermediateOutputPath>
-    <RootIntermediateOutputPath Condition="'$(RootIntermediateOutputPath)'==''">$(BaseIntermediateOutputPath)</RootIntermediateOutputPath>
+    <BinDir>$(ProjectDir)artifacts\bin\</BinDir>
+    <RootIntermediateOutputPath>$(ProjectDir)artifacts\obj\</RootIntermediateOutputPath>
```

Comment:
I introduced that property in buildtools. I don't see any usage of it in arcade. You can probably remove it?

Reply:
ArtifactsObjDir is now the new property so this isn't required anymore.

Reply:
I'm planning on removing it when I cut out buildtools altogether, for now it still gets referenced in places like `ReferenceAssemblies.targets`

Reply:
Of course only if you aren't using buildtools anymore...